timesketch/frontend-ng/src/views/Sketch.vue

```diff
@@ -105,59 +112,33 @@ limitations under the License.
           </router-link>
         </v-avatar>
 
-        <div
-          @click="showSketchMetadata = !showSketchMetadata"
-          style="font-size: 1.1em; cursor: pointer; white-space: nowrap; overflow: hidden; text-overflow: ellipsis"
-          :title="sketch.name"
-        >
-          {{ sketch.name }}
-        </div>
+        <v-hover v-slot="{ hover }">
+          <div class="d-flex flex-wrap">
+            <div
+              class="flex-1-0"
+              @dblclick="renameSketchDialog = true"
```

Comment:
Can you add the option to rename a sketch also to the top right submenu (the one with "archive sketch" etc). My observation with the team so far showed that people were looking there first.

Reply:
ah, good catch. I'll add that as well 👍🏼

Reply:
added:

<img width="397" alt="Screenshot 2023-07-04 at 14 22 33" src="https://github.com/google/timesketch/assets/316362/11c7635b-fbe5-49b9-ab86-973f34354a91">